lib/commonmarker/renderer.rb

```diff
@@ -4,13 +4,14 @@
 module CommonMarker
   class Renderer
     attr_accessor :in_tight, :warnings, :in_plain
-    def initialize
+    def initialize(options: :DEFAULT, extensions: [])
+      @opts = Config.process_options(options, :render)
       @stream = StringIO.new("".force_encoding("utf-8"))
       @need_blocksep = false
       @warnings = Set.new []
       @in_tight = false
       @in_plain = false
-      @buffer = ''
+      @tagfilter = extensions.include?(:tagfilter)
```

Comment:
It's the name of the `tagfilter` extension as bundled with `cmark-gfm`; we just detect the name of it here and emulate its behaviour.